sites/all/modules/custom/debugacademy/debugacademy.admin.inc

```diff
@@ -0,0 +1,294 @@
+<?php
+
+/**
+ * @file
+ * Admin page to update content on debugacademy theme2 homepage.
+ */
+
+/**
+ * System settings form to manage theme 2 homepage content.
+ */
+function debugacademy_admin_settings() {
+  $form['description'] = array(
+    '#markup' => '<p>' . t('Form to modify Debug Academy Theme 2 Homepage content.') . '</p>',
+  );
+
+  /*Main content on homepage*/
+  $form['main_section'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Main Section Content'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+  );
+  $form['main_section']['debugacademy_main_text'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Main Header'),
+    '#description' => t('Main text to grab attention.'),
+    '#default_value' => variable_get('debugacademy_main_text', '<h1>Graduate with a career,<br /><strong>Not loans.</strong></h1>'),
+    '#required' => TRUE,
+  );
+  
+  /*Orange hire popup*/
+  $form['main_section']['debugacademy_hire_popup'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Small side popup'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+  );  
+  $form['main_section']['debugacademy_hire_popup']['debugacademy_hire_pop_title'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Popup title'),
+    '#description' => t('Attention grabbing one-liner for side popup'),
+    '#default_value' => variable_get('debugacademy_hire_pop_title', 'Hire our next class to work on YOUR project'),
+    '#required' => TRUE,
+  );
+  $form['main_section']['debugacademy_hire_popup']['debugacademy_hire_pop_body'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Popup body'),
+    '#description' => t('Small popup on right side of main page.'),
+    '#default_value' => variable_get('debugacademy_hire_pop_body', 'Client projects are led by the teacher, a full-time senior developer.'),
+    '#required' => TRUE,
+  );
+
+  /*Student Pane Content*/
+  $form['main_section']['debugacademy_student_pane'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Student content pane'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+  );  
+  $form['main_section']['debugacademy_student_pane']['debugacademy_student_pane_line_one'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Student pane line one'),
```

Comment:
Having each line be its own input is restrictive - I think all three lines should be in one large text field @malikkotob